edge-modules/NetworkController/Mode.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    internal enum Mode
+    {
+        Offline = 0,
+        Satellite
```

Comment:
Was only setting value for Offline intentional?

Reply:
It is common to have a default value which usually is 0, here I wanted Offline to be the default so I just wanted to make it clear by setting the value to 0 which is the default